app/Http/Controllers/PrintController.php

```diff
@@ -3,16 +3,51 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Support\Facades\Storage;
 use Illuminate\Support\Facades\Validator;
 use App\User;
 use App\PrintAccount;
+use App\PrintJob;
 
 class PrintController extends Controller
 {
     public function index() {
         return view('print.app');
     }
 
+    public function print(Request $request) {
+        $validator = Validator::make($request->all(), [
+            'file_to_upload' => 'required|file|mimes:pdf|max:120000',
+            'number_of_copies' => 'required|integer|min:1'
+        ]);
+        $validator->validate();
+        
+        $print_account = Auth::user()->printAccount;
+        $two_sided = $request->has('two_sided');
+        $file = $request->file_to_upload;
+        $pages = exec("pdfinfo " . $file->getPathName() . " | grep '^Pages' | awk '{print $2}' 2>&1");
+
+        if($pages == "" || !is_numeric($pages) || $pages == 0){
```

Comment:
Please autoformat the code, to make it consistent with the rest of the codebase. Use spaces after the keywords like `if`.
For more in-depth style guide read the discussion on #37.

Reply:
Probably worth it to check for negative value, but not sure.

Reply:
Or simply hoisting out this whole check to a function, and to a different function the logging and reporting logic for this branch.

Reply:
> Please autoformat the code, to make it consistent with the rest of the codebase.

Yeah, have not set up my environment, yet. Be careful with autformat however, until we agree in a unified convention.

Reply:
> Probably worth it to check for negative value, but not sure.

Will change, just to make sure.

Reply:
> Or simply hoisting out this whole check to a function, and to a different function the logging and reporting logic for this branch.

Modified accordingly.